classes/file/TemporaryFileManager.inc.php

```diff
@@ -57,6 +57,16 @@ function deleteFile($fileId, $userId) {
 	}
 
 	/**
+	 * Delete a file by path.
+	 * For temporary files that are not in the DB, e.g. exported xml files.
+	 * @param $filePath string the location of the file to be deleted
+	 * @return boolean returns true if successful
+	 */
+	function deleteFileByPath($filePath) {
```

Comment:
Hmmm... Somehow I don't feel this is good ;-) -- I have first to create a file (from XML string) in order then to divide it apart, to fit the TemporaryFile structure, to get and save information into the DB, that I don't need at all (newFileName, fileType, fileSize, originalFileName). Thus, maybe then to:
1) write a new class (ExportFileManager?) that will extend the PrivateFileManager and just add the new base path = '/temp/'. Or to 
2) use just a normal FileManager, giving it the manually constructed path to that temp folder? -- I actually only need that base path from the TemporaryFileManager (and to mark it as a temporary file for the code readers) :-P Everything else what I need is in the FileManager (writeFile, downloadFile, deleteFile -- that's it). 
What do you think? Maybe the #2?